src/util.cpp

```diff
@@ -426,87 +426,87 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
+        "<p>dszgit (<a href=\"https://github.com/dszgit\">dszgit</a>)</p>"
+        "<p>nefarius2001 (<a href=\"https://github.com/nefarius2001\">nefarius2001</a>)</p>"
+        "<p>jc-Rosichini (<a href=\"https://github.com/jc-Rosichini\">jc-Rosichini</a>)</p>"
+        "<p>Julian Santander (<a href=\"https://github.com/j-santander\">j-santander</a>)</p>"
+        "<p>chigkim (<a href=\"https://github.com/chigkim\">chigkim</a>)</p>"
+        "<p>Bodo (<a href=\"https://github.com/bomm\">bomm</a>)</p>"
+        "<p>Christian Hoffmann (<a href=\"https://github.com/hoffie\">hoffie</a>)</p>"
+        "<p>jp8 (<a href=\"https://github.com/jp8\">jp8</a>)</p>"
+        "<p>James (<a href=\"https://github.com/jdrage\">jdrage</a>)</p>"
+        "<p>ranfdev (<a href=\"https://github.com/ranfdev\">ranfdev</a>)</p>"
+        "<p>bspeer (<a href=\"https://github.com/bspeer\">bspeer</a>)</p>"
+        "<p>Martin Passing (<a href=\"https://github.com/passing\">passing</a>)</p>"
+        "<p>DonC (<a href=\"https://github.com/dcorson-ticino-com\">dcorson</a>)</p>"
         "<br>" + tr ( "For details on the contributions check out the " ) +
-        "<a href=""https://github.com/jamulussoftware/jamulus/graphs/contributors"">" + tr ( "Github Contributors list" ) + "</a>." );
+        "<a href=\"https://github.com/jamulussoftware/jamulus/graphs/contributors\">" + tr ( "Github Contributors list" ) + "</a>." );
 
     // translators
     txvTranslation->setText (
         "<p><b>" + tr ( "Spanish" ) + "</b></p>"
-        "<p>Daryl Hanlon (<a href=""https://github.com/ignotus666"">ignotus666</a>)</p>"
+        "<p>Daryl Hanlon (<a href=\"https://github.com/ignotus666\">ignotus666</a>)</p>"
         "<p><b>" + tr ( "French" ) + "</b></p>"
-        "<p>Olivier Humbert (<a href=""https://github.com/trebmuh"">trebmuh</a>)</p>"
+        "<p>Olivier Humbert (<a href=\"https://github.com/trebmuh\">trebmuh</a>)</p>"
         "<p><b>" + tr ( "Portuguese" ) + "</b></p>"
-        "<p>Miguel de Matos (<a href=""https://github.com/Snayler"">Snayler</a>)</p>"
-        "<p>Melcon Moraes (<a href=""https://github.com/melcon"">melcon</a>)</p>"
+        "<p>Miguel de Matos (<a href=\"https://github.com/Snayler\">Snayler</a>)</p>"
+        "<p>Melcon Moraes (<a href=\"https://github.com/melcon\">melcon</a>)</p>"
         "<p><b>" + tr ( "Dutch" ) + "</b></p>"
-        "<p>Jeroen Geertzen (<a href=""https://github.com/jerogee"">jerogee</a>)</p>"
+        "<p>Jeroen Geertzen (<a href=\"https://github.com/jerogee\">jerogee</a>)</p>"
         "<p><b>" + tr ( "Italian" ) + "</b></p>"
-        "<p>Giuseppe Sapienza (<a href=""https://github.com/dzpex"">dzpex</a>)</p>"
+        "<p>Giuseppe Sapienza (<a href=\"https://github.com/dzpex\">dzpex</a>)</p>"
         "<p><b>" + tr ( "German" ) + "</b></p>"
-        "<p>Volker Fischer (<a href=""https://github.com/corrados"">corrados</a>)</p>"
+        "<p>Volker Fischer (<a href=\"https://github.com/corrados\">corrados</a>)</p>"
```

Comment:
Do we know if this is true?  Is anyone else looking to pick up the German if not (which I suspect is the case)?

Reply:
I assume this is no longer true. A german translator is missing. I don't want to do it myself since I'd like to focus on other things. I assume there are multiple other german/swiss/austrian people especially in the facebook group who could translate Jamulus. Could you please ask in the facebook group?

Reply:
I've just posted this in the FB group:
> The Jamulus development team is looking for a technical, native German speaker to maintain the German translations for the Jamulus program. The translation file already exists, so this only involves producing translations for new or updated text. Please PM me if you are interested.

Reply:
Thank you very much! If somebody PMs you (I hope so), he/she can just ping me somewhere here on GitHub or the organisation. Nevertheless, I can't help much since I'm only familiar with the translation of the wiki.